arch/tricore/include/syscall.h

```diff
@@ -171,6 +184,18 @@ static inline uintptr_t sys_call1(unsigned int nbr, uintptr_t parm1)
     : "i"(SYS_syscall)
     : "memory", "a11"
   );
+#else
+  register long reg0 __asm__("d8") = (long)(nbr);
```

Comment:
could we share the same implementation between tasking and gcc?

Reply:
No, the syntax of tasking and gnuc is incompatible